src/app/components/card/card.component.ts

```diff
@@ -144,4 +151,33 @@ export class CardComponent {
 
         return '';
     }
+
+    private calculateBaseColor(): RGB {
+        const containsTypeNight = this.card.types.some(
+            (cardType: CardType) => cardType.id === CardTypeId.Night,
+        );
+        const containsPrimaryType = this.card.types.some(
+            (cardType: CardType) =>
+                this.orderedPrimaryCardTypes.includes(cardType.id) ||
+                cardType.id === CardTypeId.Action,
```

Comment:
Could we just add `CardTypeId.Action` to `orderedPrimaryCardTypes`?

Reply:
This unfortunately leads to problem with the calculation of the background color.